### ISSUE TITLE
Disabling selector-combinator-space-before rule inline in CSS fails

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#390]
Given the following CSS:

```css
/* stylelint-disable selector-combinator-space-before */
.reordering .menu-item-depth-0  > .menu-item-bar { margin-right: 0;     }
.reordering .menu-item-depth-1  > .menu-item-bar { margin-right: 15px;  }
.reordering .menu-item-depth-2  > .menu-item-bar { margin-right: 30px;  }
.reordering .menu-item-depth-3  > .menu-item-bar { margin-right: 45px;  }
.reordering .menu-item-depth-4  > .menu-item-bar { margin-right: 60px;  }
.reordering .menu-item-depth-5  > .menu-item-bar { margin-right: 75px;  }
.reordering .menu-item-depth-6  > .menu-item-bar { margin-right: 90px;  }
.reordering .menu-item-depth-7  > .menu-item-bar { margin-right: 105px; }
.reordering .menu-item-depth-8  > .menu-item-bar { margin-right: 120px; }
.reordering .menu-item-depth-9  > .menu-item-bar { margin-right: 135px; }
.reordering .menu-item-depth-10 > .menu-item-bar { margin-right: 150px; }
.reordering .menu-item-depth-11 > .menu-item-bar { margin-right: 165px; }
/* stylelint-enable */
```
(Lines 1 - 10 above have two spaces preceding the `>` combinator to align the formatting.)

Using the following rule configuration:
```
    "selector-combinator-space-after": [ 2, "always" ],
    "selector-combinator-space-before": [ 2, "always" ],
```

The following warning is the result: (10 warnings for the 10 instances)
```
435:33	⚠  Expected single space before ">" combinator (selector-combinator-space-before) [stylelint]
436:33	⚠  Expected single space before ">" combinator (selector-combinator-space-before) [stylelint]
437:33	⚠  Expected single space before ">" combinator (selector-combinator-space-before) [stylelint]
438:33	⚠  Expected single space before ">" combinator (selector-combinator-space-before) [stylelint]
439:33	⚠  Expected single space before ">" combinator (selector-combinator-space-before) [stylelint]
440:33	⚠  Expected single space before ">" combinator (selector-combinator-space-before) [stylelint]
441:33	⚠  Expected single space before ">" combinator (selector-combinator-space-before) [stylelint]
442:33	⚠  Expected single space before ">" combinator (selector-combinator-space-before) [stylelint]
443:33	⚠  Expected single space before ">" combinator (selector-combinator-space-before) [stylelint]
444:33	⚠  Expected single space before ">" combinator (selector-combinator-space-before) [stylelint]
```

Note: The above is an excerpt of my full CSS, with my full CSS I'm actually disabling 4 rules in total:
```/* stylelint-disable block-opening-brace-newline-after, block-opening-brace-space-before, selector-combinator-space-before, block-closing-brace-newline-before */```, the only rule that is not disabled is the `selector-combinator-space-before` rule, the other 3 rules are successfully disabled.

**Comment by davidtheclark** [comment]
Oooo, this was a very good bug to have found. Thanks! Will fix soon.

**Comment by davidtheclark** [comment]
@ntwb : All of your great bugs & requests (thanks again!) should be addressed in 1.1.0, just released (except the one I didn't quite understand, #388). Please let me know if those fixes work as expected.

**Comment by ntwb** [comment]
I thought this would be a fun one :smirk: Confirming this as fixed :+1:

**Comment by ntwb** [comment]
David, thanks for your work in addressing all the bugs I submitted, much more timely and speedy than my replies here thanking you now, crazy week, it's greatly appreciated. :smile:
